### PR TITLE
Change message for no Latest resolution

### PR DESCRIPTION
**Pull request by mit-mit**
Attempt to make the message for no possible resolution for Latest versions a bit easier to read

### CODE REVIEW

**Review by jonasfj — APPROVED** [review]

**Comment by jonasfj** [comment]
It's a complicated scenario... and not easy to explain in 2 lines... We could also leave `dart.dev/go/...`-link and then explain it in an FAQ on dart.dev

**Comment by mit-mit** [comment]
That's right. But is this attempt easier to understand than the current?

File: lib/src/command/outdated.dart [hunk 1]
```diff
@@ -501,8 +501,9 @@ Future<void> _outputHuman(
       upgradable == 0 &&
       rows.isNotEmpty) {
     log.message(
-        '\nDependencies are all constrained to the latest resolvable versions.'
-        '\nNewer versions, while available, are not mutually compatible.');
+        '\nCurrent dependencies are constrained to versions listed in Resolveable.'
```
Comment (sigurdm) [inline lib/src/command/outdated.dart:504]:
```suggestion
        '\nCurrent dependencies are constrained to versions listed in Resolvable.'
```

**Comment by sigurdm** [comment]
I liked @leafpetersen 's [suggestion](https://github.com/dart-lang/pub/issues/2702#issuecomment-722706204) for using fewer hard words ("mutually compatible").

I also think it is better just to say that they are not 'mutually compatible' instead of 'mutually compatible with current dependencies'.

I don't think the 'current' versions really matter that much here.

**Comment by mit-mit** [comment]
@sigurdm what about this update (I just committed):

```
Current dependencies are constrained to versions listed in Resolvable.
A resolution compatible with the Latest versions doesn't exist, as they are not
mutually compatible.
```

**Comment by sigurdm** [comment]
I ... think it's better. Should we ask @kwalrath ?

**Comment by mit-mit** [comment]
> Should we ask @kwalrath ?

I'd like to land this now if we think it's ~better than the current, so we can start the Dart roll.

But Kathy will be testing the overall flows next week, and I'll make sure to ask her to review all UI messages.

**Review by sigurdm — APPROVED** [review]
LGTM

**Pull request merged**